packages/react-data-grid/src/common/constants/index.ts

```diff
@@ -0,0 +1,28 @@
+import * as CellNavigationMode from './CellNavigationMode';
+import * as EventTypes from './EventTypes';
+import * as HeaderRowType from './HeaderRowType';
+
+enum UpdateActions {
+  CELL_UPDATE,
```

Comment:
we need to use string enums here

Reply:
For debugging purposes?

Reply:
for example when you are debugging an cell update event, and the event is passed to somewhere else and you only have the instance. e.g. when you see `event.updateType(don't remember the correct field)`, it will give you the string value `CELL_UPDATE` rather than `0, 1, 2, 3`